src/components/adslotUi/FilePickerComponent.js

```
@@ -0,0 +1,77 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import Button from 'react-bootstrap/lib/Button';
+
+require('styles/adslotUi/FilePicker.scss');
+
+const baseClass = 'filepicker-component';
+
+class FilePickerComponent extends React.Component {
+  constructor() {
+    super();
+    this.state = { fileName: '' };
+    this.onChange = this.onChange.bind(this);
+    this.removeFile = this.removeFile.bind(this);
+  }
+
+  onChange(event) {
+    this.setState({ fileName: event.target.files[0].name });
+    this.props.onSelect(event.target.files[0]);
+  }
```

What happens when multiple files are selected? Do we ignore them, should we throw an error, or does it not let you select multiple?